dll/win32/shell32/wine/shpolicy.c

```diff
@@ -617,6 +652,7 @@ static POLICYDATA sh32_policy_table[] =
     "NoHardwareTab",
     SHELL_NO_POLICY
   },
+#if (WINE_FILEVERSION_MAJOR <= 6) && (NTDDI_VERSION < NTDDI_LONGHORN)
```

Comment:
Question to reviewers: Do you think Wine would be OK with all this extra versioning information?